testfiles/HybridModel_4Zone.idf

```diff
@@ -0,0 +1,1459 @@
+! 4ZoneHybridModel.idf
```

Comment:
1. Changed the example file name to "HybridZoneModel" as recommended. 
2. Deleted all tabs. 
3. Added a line to CMakeLists.txt for this this example file